app/assets/stylesheets/_navigation.css.scss

```diff
@@ -1,26 +1,27 @@
 // Navigation Styles
 
-.navbar-home {
-  @extend .navbar-brand;
-  font-size: inherit;
+.navbar-right > li > a {
+  padding-right: 5px;
 }
 
-nav {
-  @extend .navbar;
-  @extend .navbar-default;
-}
-
-.navbar-collapse-button {
-  @extend .navbar-toggle;
-  @extend .collapsed;
-}
+.navbar-nav > li.gravatar {
+  margin-right: -5px;
+  .round {
+    display:block;
```

Comment:
Properties should be ordered border-radius, display, height, overflow, padding, width<br>Colon after property should be followed by one space